### PR TITLE
Add chain list json

### PR DESCRIPTION
**Pull request by ERussel**

### CODE REVIEW

File: chains/chains.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  {
+    "chainId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",
+    "assets": [
+      {
+        "assetId": 0,
+        "name": "DOT",
+        "symbol": "DOT",
+        "icon": "https://raw.githubusercontent.com/soramitsu/fearless-utils/master/chains/icons/polkadot.svg",
+        "precision": 10,
+        "isUtility": true
```
Comment (antonkhvorov) [inline chains/chains.json:11]: Utility asset can be only one, so maybe we should consider making Utility asset as asset with assetId == 0?

File: chains/chains.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      }
+    ],
+    "nodes": [
+      {
+        "url": "wss://polkadot.api.onfinality.io/ws",
+        "name": "Polkadot, OnFinality Node"
+      }
+    ],
+    "icon": "https://raw.githubusercontent.com/soramitsu/fearless-utils/master/chains/icons/polkadot.svg",
+    "prefix": 0,
```
Comment (antonkhvorov) [inline chains/chains.json:21]: prefix -> addressPrefix for clarity?

File: chains/chains.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "types": "https://raw.githubusercontent.com/polkascan/py-scale-codec/master/scalecodec/type_registry/polkadot.json",
+    "isEthereumBased": false
```
Comment (antonkhvorov) [inline chains/chains.json:23]: 🤝

**Review by antonkhvorov — APPROVED** [review]

File: chains/chains.json (hunk 1 above)
Comment (ERussel) [reply chains/chains.json]: Good idea. Fixed

File: chains/chains.json (hunk 2 above)
Comment (ERussel) [reply chains/chains.json]: fixed

**Pull request merged**